File: guilherme/Problema0421/Problema0421.java
import java.util.Scanner;

public class Problema0421{
    static Scanner input = new Scanner(System.in);

    public static void primo(int n){
        boolean ehPrimo = true;
        for(int i = 2; i < n; i++){
            if(n % i == 0){
                ehPrimo = false;
            }
        } 
        if(ehPrimo == false){
            System.out.println("O número "+n+" não é primo!");
        }else{
            System.out.println("O número "+n+" é primo!");
        }
    }

    public static void main(String[] args) {
        int n;
        System.out.println("Digite um número: ");
        n = Integer.valueOf(input.nextLine());
        primo(n);
    }
}

File: guilherme/Problema0421/Problema0421.py
def primo(n):
    ehPrimo = True
    for i in range(2, n):
        if(n % i == 0):
            ehPrimo = False
    if(ehPrimo == False):
        print("O número",n,"não é primo!")
    else:
        print("O número",n,"é primo!")

n = int(input("Digite um número: "))
primo(n)